hack/OSBS/build-osbs.sh

```diff
@@ -0,0 +1,208 @@
+    set +e
+    if [ -n "$KERBEROS_PASSWORD" ]; then
+        echo "$KERBEROS_PASSWORD" | kinit "$KERBEROS_PRINCIPAL"
+        _klist
+        if [ "$?" -ne 0 ]; then
+            echo "Failed to get kerberos token for $KERBEROS_PRINCIPAL with password"
+            exit -1
+        fi
+    elif [ -n "$KERBEROS_KEYTAB" ]; then
+        kinit -k -t "$KERBEROS_KEYTAB" "$KERBEROS_PRINCIPAL"
+        _klist
+        if [ "$?" -ne 0 ]; then
+            echo "Failed to get kerberos token for $KERBEROS_PRINCIPAL with $KERBEROS_KEYTAB"
+            exit -1
+        fi
+    else
+        echo "No kerberos password or keytab specified with KERBEROS_PASSWORD or KERBEROS_KEYTAB"
+        exit -1
+    fi
+    set -e
+}
+
+# _klist will help to indentify if the kerberos ticket, prints when debug is enabled
+function _klist() {
+    if [ -n "$DEBUG" ]; then
+        klist
+    fi
+}
+
+
+function set_git_config() {
+    git config --global user.email "$GIT_EMAIL"
+    git config --global user.name  "$GIT_USER"
+    git config --global core.pager ""
+}
+
+
+while getopts gu:e:v:c:t:o:r:n:d:p:k:s:b:l:i:w:h option; do
+    case $option in
+        g)
+            DEBUG=true
+            ;;
+        u)
+            GIT_USER=$OPTARG
+            ;;
+        e)
+            GIT_EMAIL=$OPTARG
+            ;;
+        v)
+            PROD_VERSION=$OPTARG
+            ;;
+        c)
+            PROD_COMPONENT=$OPTARG
+            ;;
+        t)
+            OSBS_BUILD_TARGET=$OPTARG
+            ;;
+        o)
+            CEKIT_BUILD_OPTIONS=$OPTARG
+            ;;
+        p)
+            KERBEROS_PRINCIPAL=$OPTARG
+            ;;
+        k)
+            KERBEROS_KEYTAB=$OPTARG
+            ;;
+        s)
+            KERBEROS_PASSWORD=$OPTARG
+            ;;
+        b)
+            BUILD_DATE=$OPTARG
+            ;;
+        l)
+            CEKIT_CACHE_LOCAL=$OPTARG
+            ;;
+        i)
+            OSBS_BUILD_USER=$OPTARG
+            ;;
+        w)
+            WORK_DIR=$OPTARG
+            ;;
+        h)
+            help
+            exit 0
+            ;;
+        *)
+            ;;
+    esac
+done
+shift $((OPTIND-1))
+
+mkdir -p $WORK_DIR
+bo_options=" --no-color"
+
+check_for_required_envs
+set_git_config
+
+if [ -n "$KERBEROS_PRINCIPAL" ]; then
+    get_kerb_ticket
+    # overrides the OSBS_BUILD_USER if it is not set and KERBEROS principal is in use
+    if [ ! -n "$OSBS_BUILD_USER" ]; then
+      echo "setting OSBS_BUILD_USER to KERBEROS_PRINCIPAL"
+      # need to catch only the first part of the principal, before the / otherwise 'rhpkg' will fail
+      OSBS_BUILD_USER=$(echo ${KERBEROS_PRINCIPAL} | awk -F"/" '{print $1}')
+    fi
+else
+    echo No kerberos principal specified, assuming there is a current kerberos ticket
+fi
+
+debug=
+if [ -n "$DEBUG" ]; then
+    debug="--verbose"
+fi
+
+builduser=
+if [ -n "$OSBS_BUILD_USER" ]; then
+    builduser="$OSBS_BUILD_USER"
+fi
+
+cd ../
+# Building the operator
+# TODO: WHICH target ?
```

Comment:
@rgdoliveira might know.

Reply:
@spolti sorry, I didn't understand the question properly. What target are you asking about?

Reply:
Fixed